controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -365,14 +362,10 @@ func (b *builder) setFrontendConfig(fconf *config.FrontendConfig) {
 		if b.desired.Agent.EBPF.Advanced != nil {
 			if v, ok := b.desired.Agent.EBPF.Advanced.Env[ebpf.EnvDedupeJustMark]; ok {
 				dedupJustMark, _ = strconv.ParseBool(v)
-			} else {
-				dedupJustMark, _ = strconv.ParseBool(ebpf.DedupeJustMarkDefault)
 			}
 
 			if v, ok := b.desired.Agent.EBPF.Advanced.Env[ebpf.EnvDedupeMerge]; ok {
 				dedupMerge, _ = strconv.ParseBool(v)
```

Comment:
not related to recent changes but why we ignore errors from `ParseBool` ?

Reply:
These booleans are constants from `agent_controller.go`. We are not supposed to have errors here right ?
If you prefer we can make reconcile loop crash :thinking:

Reply:
what do u mean ParseBool is what we will get from FC object for the constants u are referring to someone could set the env var to something bad which will trigger
https://cs.opensource.google/go/go/+/refs/tags/go1.21.6:src/strconv/atob.go;l=17
specially we have no API validation for this envvar try setting to "foo" and see what will happen :)

Reply:
oh I thought we were talking about defaults values here; that's right, I should manage parse error from `Env` input !

Reply:
https://github.com/netobserv/network-observability-operator/pull/529/commits/c891c2f47b81fff1abb4613fda0bd3ef57e60512